client/src/app/course.service.ts

```diff
@@ -39,7 +39,7 @@ export class CourseService {
       };
 
       this.allCourses = this.http.get<CourseData[]>(
-        `${this.uri}/course/` + school.toLowerCase(),
+        `${this.uri}/course?school=` + school.toUpperCase(),
```

Comment:
At this point I'm assuming that school will always be saved in our DB in uppercase.

Reply:
Yes, school will be SJSU for now